src/main/java/org/broadinstitute/hellbender/tools/walkers/sv/JointGermlineCNVSegmentation.java

```diff
@@ -0,0 +1,483 @@
+
+    private SortedSet<String> samples;
+    private VariantContextWriter vcfWriter;
+    private SAMSequenceDictionary dictionary;
+    private SVDepthOnlyCallDefragmenter defragmenter;
+    private SVClusterEngine clusterEngine;
+    private List<GenomeLoc> callIntervals;
+    private String currentContig;
+    private SampleDB sampleDB;
+    private boolean doDefragmentation;
+
+    protected final OneShotLogger oneShotLogger = new OneShotLogger(logger);
+
+    public static final String MIN_QUALITY_LONG_NAME = "minimum-qs-score";
+    public static final String MODEL_CALL_INTERVALS = "model-call-intervals";
+    public static final String BREAKPOINT_SUMMARY_STRATEGY = "breakpoint-summary-strategy";
+
+    @Argument(fullName = MIN_QUALITY_LONG_NAME, doc = "Minimum QS score to combine a variant segment")
+    private int minQS = 20;
+
+    @Argument(fullName = MODEL_CALL_INTERVALS, doc = "Intervals used for gCNV calls.  Should be preprocessed and filtered to line up with model calls. Required for exomes.")
+    private File modelCallIntervalList;
+
+    @Argument(fullName = BREAKPOINT_SUMMARY_STRATEGY, doc = "Strategy to use for choosing a representative value for a breakpoint cluster.")
+    private SVClusterEngine.BreakpointSummaryStrategy breakpointSummaryStrategy = SVClusterEngine.BreakpointSummaryStrategy.MEDIAN_START_MEDIAN_END;
+
+    @Argument(fullName= StandardArgumentDefinitions.OUTPUT_LONG_NAME,
+            shortName=StandardArgumentDefinitions.OUTPUT_SHORT_NAME,
+            doc="The combined output file", optional=false)
+    private File outputFile;
+
+    @Argument(
+            doc = "Reference copy-number on autosomal intervals.",
+            fullName = PostprocessGermlineCNVCalls.AUTOSOMAL_REF_COPY_NUMBER_LONG_NAME,
+            minValue = 0,
+            optional = true
+    )
+    private int refAutosomalCopyNumber = 2;
+
+    @Argument(
+            doc = "Contigs to treat as allosomal (i.e. choose their reference copy-number allele according to " +
+                    "the sample karyotype).",
+            fullName = PostprocessGermlineCNVCalls.ALLOSOMAL_CONTIG_LONG_NAME,
+            optional = true
+    )
+    private List<String> allosomalContigList = Arrays.asList("X","Y","chrX","chrY");
+
+    /**
+     * See https://software.broadinstitute.org/gatk/documentation/article.php?id=7696 for more details on the PED
+     * format. Note that each -ped argument can be tagged with NO_FAMILY_ID, NO_PARENTS, NO_SEX, NO_PHENOTYPE to
+     * tell the GATK PED parser that the corresponding fields are missing from the ped file.
+     *
+     */
+    @Argument(fullName=StandardArgumentDefinitions.PEDIGREE_FILE_LONG_NAME, shortName=StandardArgumentDefinitions.PEDIGREE_FILE_SHORT_NAME, doc="Pedigree file for samples", optional=false)
+    private File pedigreeFile = null;
+
+    @Override
+    public boolean doDictionaryCrossValidation() {
+        return false;
```

Comment:
This is to avoid the inefficient dictionary checks mentioned above.  The tool requires a reference, though, so each input VCF will be checked against the reference, which is just O(N).